documentation/API-design-guidelines.md

```diff
@@ -1287,61 +1302,84 @@ response:
-| eventDetail | object | Event details structure depending on the eventType | mandatory |
+| eventSubscriptionId | string | The event subscription identifier - must be valued for Resource-based subscription | optional |
+| ... | ... | Specific attribute(s) related to the notification event | ... |
+
 
 Note: For operational and troubleshooting purposes it is relevant to accommodate use of `X-Correlator` header attribute. API listener implementations have to be ready to support and receive this data.
 
-Specific eventType "SUBSCRIPTION_ENDS" is defined to inform listener about subscrition termination. It is used when the subscription expire time (required by the requester) has been reached or if the API server has to stop sending notification prematurely. For this specific event, the `eventDetail` must feature `terminationReason` attribute.
+Specific event notification type "subscription-ends" is defined to inform listener about subscrition termination. It is used when the subscription expire time (required by the requester) has been reached or if the API server has to stop sending notification prematurely. For this specific event, the `data` must feature `terminationReason` attribute.
 
-_Error definition for event notification_
+#### Error definition for event notification
 
 Error definition are described in this guideline applies for event notification.
 
 Following Error code must be present:
 * for `POST`: 400, 401, 403, 500, 503
 
-_Managing correlation between resource-based event souscription and event notification_
+#### Correlation Management
 To manage correlation between the subscription management and the event notification (as these are 2 distinct operations):
-- use `eventSubscriptionId` attribute (in body) - this identifier is provided in event subscription and could be valued in each event notification. 
+- use `eventSubscriptionId` attribute (in `data` structure in the body) - this identifier is provided in event subscription and could be valued in each event notification. 
+
+Note: There is no normative enforcement to use any of these patterns and they could be used on agreement between API consumer & provider.
+
+#### Security Considerations
+
+As notification may carry sensitive information, privacy and security constraints has to be considered. CloudEvents specification provides some guidance there: https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#privacy-and-security
+
+#### Abuse Protection
+
+Any system that allows registration of and delivery of notifications to arbitrary HTTP endpoints can potentially be abused such that someone maliciously or inadvertently registers the address of a system that does not expect such requests and for which the registering party is not authorized to perform such a registration.
+
+In order to protect the sender, CloudEvents specification provides some guidance there: https://github.com/cloudevents/spec/blob/main/cloudevents/http-webhook.md#4-abuse-protection
 
```

Comment:
Event Producers shall choose based on their internal security guidelines to implement measures based on the above guidance to ensure abuse protection. For e.g. An event producer might ask the subscriber to pre-register the notification URL at the time of app onboarding. If this registered notification URL doesn't match later with the notification URL in the request, the event producer can choose to reject the request with the relevant error code.

Reply:
Make sense to add this paragraph. I've added it. Thanks !